src/ORM/Rule/ExistsIn.php

```diff
@@ -40,14 +40,28 @@ class ExistsIn
     protected $_repository;
 
     /**
+     * Options for the constructor
+     *
+     * @var array
+     */
+    protected $_options = [];
```

Comment:
@markstory Stickler shouldn't be bitching about underscore prefix for protected methods and properties :)  Our CS allows it.